website/source/docs/builders/profitbricks.html.md

```diff
@@ -0,0 +1,75 @@
+---
+description: |
+    The ProfitBricks builder is able to create images for ProfitBricks cloud.
+layout: docs
+page_title: ProfitBricks Builder
+...
+
+# ProfitBricks Builder
+
+Type: `profitbricks`
+
+The ProfitBricks Builder is able to create virtual machines for [ProfitBricks](https://www.profitbricks.com).
+
+-> **Note:** This builder is not supported by HashiCorp. Please visit [ProfitBricks DevOps Central](https://devops.profitbricks.com/) for support. You may file issues on [GitHub](https://github.com/profitbricks/docker-machine-driver-profitbricks/issues).
+
+## Configuration Reference
+
+There are many configuration options available for the builder. They are
+segmented below into two categories: required and optional parameters. Within
+each category, the available configuration keys are alphabetized.
+
+In addition to the options listed here, a
+[communicator](/docs/templates/communicator.html) can be configured for this
+builder.
+
+### Required
+
+-   `username` (string) - ProfitBricks username. This can be specified via environment variable `PROFITBRICKS_USERNAME', if provided. The value definded in the config has precedence over environemnt variable. 
+
+-   `password` (string) - ProfitBrucks password. This can be specified via environment variable `PROFITBRICKS_PASSWORD', if provided. The value definded in the config has precedence over environemnt variable.
+
+-   `image` (string) - ProfitBricks volume image 
+
+### Optional
+
+-   `cores` (int) - Amount of CPU cores to use for this build. Defaults to `4`.
```

Comment:
int -> integer, in several places.
Default values should have be double quoted `4` -> "4", in several places.